easybuild/tools/filetools.py

```diff
@@ -1095,7 +1095,7 @@ def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs
             if relative:
```

Comment:
I would suggest a
if not os.path.islink(path):
before the if relative:
covering the whole if-else of the relative:
Since if skip_symlinks is False the chmod should not be done at all.

Reply:
But that would make `skip_symlinks` effectively meaningless, since `adjust_permissions` would then always ignore symlinks?

Reply:
No, it would still change group, but never touch symlinks, and for security reason one should never do chmod on a symlink, unless it's been verified that it stays within the intended tree.
(And that it haven't moved between checking that and the actual chmod, which is basically impossible)

Reply:
But yes, ignoring symlinks is what you should always do when it comes to chmod, and for owner/group it really doesn't matter that much...